### PR TITLE
DOCSP-41216 - Broken link

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41216
Staging - https://preview-mongodbmongokart.gatsbyjs.io/cpp-driver/docsp-41216-broken-link/configuration/#compressing-data-to-and-from-mongodb

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/configuration.txt [hunk 1]
```diff
@@ -204,4 +204,4 @@ MongoDB 3.4 added Snappy compression support, while zlib compression was added
 in 3.6, and zstd compression in 4.2.
 
 Data compression can be enabled with the appropriate URI options, as documented
```
Comment (rustagir) [inline source/configuration.txt:206]:
```suggestion
You can enable data compression the appropriate URI options. Learn about these options
```

File: source/configuration.txt [hunk 2]
```diff
@@ -204,4 +204,4 @@ MongoDB 3.4 added Snappy compression support, while zlib compression was added
...
-in the `C driver manual <https://mongoc.org/libmongoc/current/data-compression.html>`__.
+in the `API documentation for the MongoDB C driver <https://mongoc.org/libmongoc/current/mongoc_uri_t.html#connection-options>`__.
```
Comment (rustagir) [inline source/configuration.txt:207]:
```suggestion
in the `MongoDB C driver API documentation <https://mongoc.org/libmongoc/current/mongoc_uri_t.html#connection-options>`__.
```

**Review by rustagir — APPROVED** [review]
small enough change that i dont need another look

File: source/configuration.txt (hunk 1 above)
Comment (mongoKart) [reply source/configuration.txt:206]: yeah, left the text alone because this page is currently getting redone as part of standardization. can fix though

**Pull request merged**